src/Jackett.Common/Indexers/BitHdtv.cs

```diff
@@ -110,9 +110,9 @@ public override async Task<IndexerConfigurationStatus> ApplyConfiguration(JToken
                 var dom = parser.ParseDocument(response.Content);
                 var messageEl = dom.QuerySelectorAll("table.detail td.text").Last();
                 foreach (var child in messageEl.QuerySelectorAll("a"))
-                    messageEl.RemoveChild(child);
+                    child.Remove();
```

Comment:
@cadatoiva I hope we are not breaking anything removing elements while iterating. This is not safe in other languages.

Reply:
This is the recommended way to remove elements according to the AngleSharp website:

https://anglesharp.github.io/docs/Questions.html#how-can-i-remove-all-elements-matching-a-certain-selector